### ISSUE TITLE
Show a better error message when using import instead of require

### ISSUE THREAD

**Issue by jlin412** [TheBrainFamily/cypress-cucumber-preprocessor#20]
syntaxError: 'import' and 'export' may only appear at the top level (8:0) while parsing /Users/jason.lin/cypress-cucumber-example/cypress/integration/login.feature while parsing file: /Users/jason.lin/cypress-cucumber-example/cypress/integration/login.feature

``` step definition file:
import { loginPage } from '../page-objects/login.po.js';

when('I login with good credential', () => {
	return loginPage.login('jason.lin@aetion.com', '412Debby1');
})
```

**Comment by lgandecki** [comment]
Hey @jlin412 The way we put the files together use require, import/export is not native in nodejs (or browsers for that matter) anyway :) 
We should update the docs to mention that. 
In the future we could possibly transpile the imports to requires, but not sure how necessary  or high priority is that (as always, PRs are welcome :) )

**Comment by jlin412** [comment]
I am using es6 so I am using import.
If I use require, it will work?  I will give a shot.

**Comment by lgandecki** [comment]
For now we should detect the import/export and throw a better error message, saying that "import is currently not supported, please use require". Help appreciated if someone wants to take a stab :)

**Comment by SaltyFurbreeze** [comment]
Is this still an issue? Kinda new to github but I'd like to contribute.

**Comment by lgandecki** [comment]
imports/exports fixed in 1.2.0  / #72 ! 
Sorry @SaltyFurbreeze :-) There are more to get done if you like to help. 
Ping me if you are still interested in doing some coding - it'd be great if you'd first cloned the repo and got it to the point where you can break one of the tests by changing the code.
